Header header

# Name of payload in MoveIt! and workspace
string name

# Current pose of payload
geometry_msgs/Pose pose

# Confidence of pose estimate. Between 0 and 1
float64 confidence

---

bool success
